src/main/groovy/edu/oregonstate/mist/directoryapi/DirectoryApplication.groovy

```diff
@@ -17,15 +21,37 @@ class DirectoryApplication extends Application<DirectoryApplicationConfiguration
     public void run(DirectoryApplicationConfiguration configuration, Environment environment) {
         this.setup(configuration, environment)
 
+        def ldapConnectionPool = configureLdapPool(configuration.ldapConfiguration)
+
         final DirectoryEntityDAO DIRECTORYENTITYDAO = new DirectoryEntityDAO(
                 configuration.ldapConfiguration,
+                ldapConnectionPool
         )
 
         def endpointUri = configuration.api.endpointUri
         environment.healthChecks().register('LDAP', new LDAPHealthCheck(DIRECTORYENTITYDAO))
         environment.jersey().register(new DirectoryEntityResource(DIRECTORYENTITYDAO, endpointUri))
     }
 
+    private static configureLdapPool(Map<String,Object> ldapConfiguration) {
+        def ldapURL = (String) ldapConfiguration.get('url')
```

Comment:
Small nitpick, but this line "`(Object) ldapConfiguration.get('foo')`" has a space while the ones in `poolConfig` don't "`(Object)ldapConfiguration.get('foo')`"

Reply:
Fixed.